Added the Get File Sample - Generic playbook (#24488)
demisto/content: Packs/CommonPlaybooks/Playbooks/Get_File_Sample_-_Generic_README.md

Retrieve files from endpoints by the file hash or the file path.

## Dependencies
This playbook uses the following sub-playbooks, integrations, and scripts.

### Sub-playbooks
* Get File Sample From Path - Generic V3
* Unzip File
* Get File Sample By Hash - Generic v3

### Integrations
This playbook does not use any integrations.

### Scripts
This playbook does not use any scripts.

### Commands
This playbook does not use any commands.

## Playbook Inputs
---

| **Name** | **Description** | **Default Value** | **Required** |
| --- | --- | --- | --- |
| MD5 | The MD5 hash of the file to retrieve. | File.MD5 | Optional |
| SHA256 | The SHA256 hash of the file to retrieve. | File.SHA256 | Optional |

> | Hostname | The hostname from which to retrieve the f ile. | Endpoint.Hostname | Optional |

Reviewer: ```suggestion
| Hostname | The hostname from which to retrieve the file. | Endpoint.Hostname | Optional |
```

> | AgentID | The Agent ID that should be used to retrieve the file. | incident.agentid | Optional |

Reviewer: ```suggestion
| AgentID | The agent ID that should be used to retrieve the file. | incident.agentid | Optional |
```

> | FilePath |  | incident.filepath | Optional |

Reviewer: ```suggestion
| FilePath | The full path to the file that needs to be retrieved. | incident.filepath | Optional |
```

> | NewFilename | Optional - a new name for the retrieved file. If left empty, the filename will not change. |  | Optional |

Reviewer: ```suggestion
| NewFilename | Optional. A new name for the retrieved file. If left empty, the filename will not change. |  | Optional |
```

| UnzipFile | Whether to unzip zipped files after retrieving them. |  | Optional |
| ZipTool | The tool used to unzip the file. The options are: 7z, zipfile. |  | Optional |
| ZipPassword | The password used to unzip zipped files. |  | Optional |

## Playbook Outputs
---

| **Path** | **Description** | **Type** |
| --- | --- | --- |

> | File | File objects - includes the zipped file and any unzipped files. | unknown |

Reviewer: ```suggestion
| File | File objects. Includes the zipped file and any unzipped files. | unknown |
```
